static/configeditor/index.html

```diff
@@ -1131,24 +1131,25 @@ <h3 id="openshift-config-editor-deploy">Deploying cartridges to ACD installation
 
   <div class="highlight"><pre class="highlight shell tab-shell">
 <code>
-  - name: "com_ibm_watson_health_car_acd_host_3_label"
+  - name: "com_ibm_watson_health_car_acd_host_&lt;n&gt;_label"
     value: "Annotator for Clinical Data - &lt;identifier&gt;"
-  - name: "com_ibm_watson_health_car_acd_host_3_url"
+  - name: "com_ibm_watson_health_car_acd_host_&lt;n&gt;_url"
     value: "https://&lt;hostname&gt;/services/clniical_data_annotator/api"
-  - name: "com_ibm_watson_health_car_acd_host_3_proxy"
+  - name: "com_ibm_watson_health_car_acd_host__&lt;n&gt;_proxy"
     value: "&lt;identifier&gt;"
-  - name: "com_ibm_watson_health_car_acd_host_3_auth"
+  - name: "com_ibm_watson_health_car_acd_host__&lt;n&gt;_auth"
     value: "Bearer"
-  - name: "com_ibm_watson_health_car_acd_host_3_phi"
+  - name: "com_ibm_watson_health_car_acd_host__&lt;n&gt;_phi"
     value: "true"
 </code></div>
 
  <ul>
   <li>where <code style="color:black;background-color:whitesmoke">&lt;identifier&gt;</code> is the identifier of the custom ACD instance, e.g. <code style="color:black;background-color:whitesmoke">custom-acd</code></li>
   <li>where <code style="color:black;background-color:whitesmoke">&lt;hostname&gt;</code> is the hostname of the custom ACD instance</li>
+  <li>where <code style="color:black;background-color:whitesmoke">&lt;n&gt;</code> is the ordinal value of the custom ACD instance starting with 1</li>
```

Comment:
Hi Dan, just a question - why did the old instructions say the index had to start at 3? Just curious. Never understood that or dug into it.

Reply:
There were multiple deployment targets configured in IBM Cloud so when this was first setup for EAP they added one and documented it - but it was wrong.  Config Editor won't see any of them unless there is a configuration with and index of 1.

Reply:
From Jennifer: host 1 and 2 were us-south and us-east.